---
layout: page
---

## Research overview

My main goal is to understand why organisms adapt to their environments in such diverse and remarkable ways.

Unfortunately, achieving this goal is problematic: most evolution has occurred in the past, and ongoing evolution typically takes longer than a human lifetime to observe. Therefore, answers to very basic questions -- why do some organisms have sex? why do some outcross and others self-fertilise? why are some haploid and others diploid? does adaptation rely on existing variation or new mutations? -- remain elusive. 

Fortunately, we have tools to overcome this: a century of mathematical models of evolution, allied with new genomic datasets and experimental systems where we can manipulate and observe evolution. 

My research uses these tools to better understand evolutionary adaptation. Examples of specific topics I have worked on include...

### Genomics of sex-specific adaptation
A major focus of my research has been to understand how one of the most dramatic forms of adaptation-—sexual dimorphism-—evolves, despite males and females sharing a near-identical genome. To this end, I have studied sexually antagonistic variants, which are a type of genetic variant that arises due to constraints on the evolution of sexual dimorphism. I identified sexually antagonistic variants in the Drosophila melanogaster genome using a genome-wide association study, described the biological functions of associated genes, and used population genomic analyses to examine patterns of polymorphisms associated with these genes using genomes of globally distributed Drosophila melanogaster populations and those of related species ([Ruzicka et al. 2019](/assets/Ruzickaetal2019PlosBiol.pdf)). 
I also helped develop new theory and methods for detecting sexually antagonistic variation. I then applied these methods to look for signals of sexually antagonistic selection flycatcher, pipefish, and human genomes ([Ruzicka et al. 2020](/assets/Ruzickaetal2020EvolLett.pdf), [Ruzicka et al. 2022](/assets/RuzickaandConnallon2022ProcB.pdf)). 

### Evolution and adaptation on sex chromosomes
Sex chromosomes provide a unique opportunity to study evolution because of their asymmetric transmission between the sexes. Using contrasts of X chromosomes and autosomes, I have examined the evolution of deleterious mutations ([Ruzicka et al. 2021](/assets/Ruzickaetal2021Genetics.pdf)), the genomic distribution of sexually antagonistic variation ([Ruzicka and Connallon 2020](/assets/RuzickaandConnallon2020ProcB.pdf), [Ruzicka and Connallon 2022](/assets/RuzickaandConnallon2022ProcB.pdf)), and the genomic distribution of inversions that capture locally-adaptive alleles ([Connallon et al. 2018](/assets/Connallonetal2018PhilTrans.pdf)).

### Linking formal theory with data
In evolutionary biology, the connections between theory and data are often loose (e.g., empiricists often rely on verbal models; theoreticians do not always generate empirically testable models; see Haller et al. 2014 and Fitzpatrick et al. 2018, *BioScience*). A key signature of my research is close integration of formal mathematical models with data analyses. For example, I helped develop models for metrics of genetic variation on autosomes and the X chromosome, and applied these metrics to human and fruit fly genomic datasets ([Ruzicka et al. 2021](/assets/Ruzickaetal2021Genetics.pdf), [Ruzicka and Connallon 2022](/assets/RuzickaandConnallon2020ProcB.pdf)). 



